src/main/kotlin/dev/arbjerg/ukulele/audio/TrackQueue.kt

```diff
@@ -13,10 +13,8 @@ class TrackQueue {
     fun peek() = queue.firstOrNull()
 
     fun removeRange(range: IntRange): List<AudioTrack> {
-        val list = queue.subList(range.first, range.last)
+        val list = queue.slice(range)
```

Comment:
`subList()` causes a `java.util.ConcurrentModificationException`. Using slice, since it makes a copy, which is what we want.